recipes/django_json_widget/meta.yaml

```diff
@@ -0,0 +1,49 @@
+
+{% set name = "django-json-widget" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ec948a4c379687f9c854748ba540fdbeb1b1730846ca953f186818521de7e696
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
```

Comment:
Can this be noarch?

Ref: https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

Reply:
I belive it can be. I'll update.